src/app/layout.tsx

```diff
@@ -43,6 +44,7 @@ export default function RootLayout({
         </main>
 
         <Footer />
+        <ScreenSizeIndicator />
```

Comment:
Could we perform this check earlier - before component definition?

```suggestion
                {process.env.NODE_ENV === 'development' && <ScreenSizeIndicator />}
```